src/main/java/uk/gov/hmcts/reform/cpo/security/AuthError.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.cpo.security;
+
+/**
+ * Authentication and Authorisation error messages.
+ **/
+public final class AuthError {
+
+    public static final String AUTHENTICATION_TOKEN_INVALID
+        = "Authentication failure due to invalid / expired tokens (IDAM / S2S).";
```

Comment:
minor:
wouldn't it help issues investigations if we could provide a distinct error messages for the IDAM token and S2S token?

Reply:
The S2S and IDAM token verification config and swagger has been 'inherited' from the MCA repository.  The only current planned divergence from this is: 

* The S2S verification steps will be enhanced during [CPO-10 _"Implement microservice based authorisation"_](https://tools.hmcts.net/jira/browse/CPO-10).
* Currently both the Create and Update endpoints will throw a 400 error (`IdAMIdCannotBeRetrievedException`: "Idam ID cannot be retrieved") if the User info cannot be loaded from IDAM, i.e. when the created_by field is populated with Used ID.

As the final required token verification behaviour would be needed across all APIs can I suggest that this behaviour is documented under new ACs in either a new ticket or CPO-10: whichever feels more relevant.